app/app.go

```diff
@@ -493,17 +530,21 @@ func NewSifApp(
 		govtypes.ModuleName,
 		minttypes.ModuleName,
 		crisistypes.ModuleName,
-		ibchost.ModuleName,
 		genutiltypes.ModuleName,
 		evidencetypes.ModuleName,
-		ibctransfertypes.ModuleName,
 		feegrant.ModuleName,
+		paramstypes.ModuleName,
+		upgradetypes.ModuleName,
+		vestingtypes.ModuleName,
+		ibchost.ModuleName,
+		disptypes.ModuleName,
+		transferModule.Name(),
 		clptypes.ModuleName,
+		ethbridgetypes.ModuleName,
+		tokenregistrytypes.ModuleName,
 		oracletypes.ModuleName,
-		ethbridge.ModuleName,
-		dispensation.ModuleName,
-		tokenregistry.ModuleName,
```

Comment:
What's the reason for removing these modules from the InitGenesis order?

Reply:
They are still there: under `ethbridgetypes.ModuleName`, `disptypes.ModuleName`, and `tokenregistrytypes.ModuleName`.

`ibctransfertypes.ModuleName` is still there too, as `transferModule.Name()`